src/mod_vcard_mnesia.erl

```diff
@@ -15,6 +16,25 @@
 -include("jlib.hrl").
 -include("mod_vcard.hrl").
 
+%%--------------------------------------------------------------------
+%% gdpr callbacks
+%%--------------------------------------------------------------------
+
+-export([get_personal_data/2]).
+
+-spec get_personal_data(gdpr:username(), gdpr:domain()) ->
+    [{gdpr:binary_table_name(), gdpr:schema(), gdpr:entities()}].
+
+get_personal_data(Username, Server) ->
```

Comment:
Why not to just use get_vcard from the main module?
Why do we need get_personal_data/2 implemented by each backend module???